test/client_and_receiver_integration_test.exs

```diff
@@ -580,7 +580,7 @@ defmodule ClientAndReceiverIntegrationTest do
     @tag client_cert: "tls/server/server_certificate.pem"
     @tag keyfile: "tls/server/private_key.pem"
 
-    @tag reason: [:handshake_failure]
+    @tag reason: [:unsupported_certificate]
```

Comment:
this test is failing on my local. I also reverted this change. It does not return an error

Reply:
I think erlang version should be `26.2.5.6`, test worked after I changed to this version

Reply:
it should be 😄